data/provider/src/main/java/tech/pegasys/artemis/api/schema/Fork.java

```diff
@@ -11,18 +11,19 @@
  * specific language governing permissions and limitations under the License.
  */
 
-package tech.pegasys.artemis.provider;
+package tech.pegasys.artemis.api.schema;
 
-import com.fasterxml.jackson.core.JsonGenerator;
-import com.fasterxml.jackson.databind.JsonSerializer;
-import com.fasterxml.jackson.databind.SerializerProvider;
-import java.io.IOException;
-import tech.pegasys.artemis.util.bls.BLSPublicKey;
+import com.google.common.primitives.UnsignedLong;
+import tech.pegasys.artemis.util.SSZTypes.Bytes4;
 
-public class BLSPublicKeySerializer extends JsonSerializer<BLSPublicKey> {
```

Comment:
BLSPublicKeySerializer renamed as Fork is confusing

Reply:
wow, that's not what happened at all, one got deleted and one got added... not sure why it'd think it's renamed...